packages/grafana-ui/src/components/uPlot/config/UPlotSeriesBuilder.ts

```diff
@@ -90,28 +96,39 @@ export class UPlotSeriesBuilder extends PlotConfigBuilder<SeriesProps, Series> {
     };
   }
 
-  getFill(): Series.Fill | undefined {
-    const { lineColor, fillColor, fillGradient, fillOpacity } = this.props;
+  private getLineColor(): Series.Stroke {
+    const { lineColor, gradientMode, colorMode, thresholds } = this.props;
 
-    if (fillColor) {
-      return fillColor;
+    if (gradientMode === GraphGradientMode.Scheme) {
```

Comment:
Can't wait to see this in action

Reply:
![Screenshot from 2021-01-18 16-38-15](https://user-images.githubusercontent.com/10999/104935936-267e7200-59ac-11eb-9b1e-23f9cb9e4cc4.png)

Reply:
![image](https://user-images.githubusercontent.com/10999/104935976-34cc8e00-59ac-11eb-9b99-6a92451f00a3.png)